unified-runtime/scripts/benchmarks/main.py

```diff
@@ -155,6 +154,9 @@ def main(directory, additional_env_vars, save_name, compare_names, filter):
         else []
     )
 
+    if not options.ur_adapter == "cuda":
+        suites.append(ComputeBench(directory))
+
```

Comment:
llama.cpp benchmark probably also needs a slight touchup to support cuda, so you should probably move it under this if.

But more generally, I think a simpler approach would be to simply return an empty benchmarks list if the adapter is unsupported, like here:
https://github.com/intel/llvm/blob/sycl/unified-runtime/scripts/benchmarks/benches/compute.py#L58

Reply:
Good point, I changed it to that.